### PR TITLE
Fixed height + color selection code, made bar's overlap 

### PR DESCRIPTION
**Pull request by saketh-n**
(given that this is probably how we may detect seizure's in practice

### CODE REVIEW

File: src/components/main.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ export default function Header() {
   const [result, setResult] = useState(null);
   const [file, setFile] = useState(null);
 
+  // Above this value we should consider this a seizure
+  const predThreshold = 70;
```
Comment (minneker) [inline src/components/main.js]: This is a great addition! After we do some more work with our dataset/model we can figure out a data-driven way to set this value as well!

File: src/components/main.js [hunk 2]
```diff
@@ -25,29 +28,52 @@ export default function Header() {
...
-      // TODO: Update to full gradient
-      let barcolor = "green-200";
-      if (res > 0.25 && res < 0.5) {
-        barcolor = "green-400";
-      } else if (res >= 0.5 && res < 0.75) {
-        barcolor = "red-200";
-      } else if (res > 0.75) {
-        barcolor = "red-400";
+
+      const red = [238, 75, 43];
```
Comment (minneker) [inline src/components/main.js]: Maybe we could refactor colors to constants file

**Review by minneker — APPROVED** [review]
lgtm! Small refactor comment, otherwise really like the gradient color scheme for the confidence bars.

File: src/components/main.js (hunk 2 above)
Comment (saketh-n) [reply src/components/main.js]: sure

**Pull request merged**